src/routes/Sensors/index.tsx

```diff
@@ -37,11 +39,27 @@ const RoundedButton = styled(Button)(() => ({
   borderRadius: '50px',
 }));
 
+function findLastSensorPosition(data: SensorData[]): Location | null {
```

Comment:
just making sure this also gets filtered by nodeId when one is selected, correct?
since you already order the datapoints, could you just fetch the first element of the ordered list we already have?

Reply:
i am just getting the last piece of information. Nodes are attached to the spotter, and there only one location for all nodes. Do we want to filter by nodeId just in case?

Reply:
hmm, ok, let's keep it like that for now. Maybe we will add nodeId or row specific buttons later on